packages/fxa-js-client/Gruntfile.js

```diff
@@ -30,10 +30,4 @@ module.exports = function(grunt) {
   grunt.registerTask('dev', ['watch:dev']);
 
   grunt.registerTask('debug', ['watch:debug']);
-
-  grunt.registerTask('travis', 'Test runner task for Travis CI', [
-    'build',
-    'intern:node',
```

Comment:
Are there stale intern subtasks that can be removed because of this?

Reply:
Intern hasn't been a dependency for quite sometime, it looks like @pdehaan edited this file about 6 years ago. We now use mocha as you can see in the `npm test` script. I think we are safe to remove `internjs` all together from this package.